clients/admin-ui/src/features/common/nav/v2/routes.ts

```diff
@@ -32,10 +32,12 @@ export const PRIVACY_NOTICES_ROUTE = "/consent/privacy-notices";
 export const CONFIGURE_CONSENT_ROUTE = "/consent/configure";
 export const ADD_MULTIPLE_VENDORS_ROUTE = "/consent/configure/add-vendors";
 export const CONSENT_REPORTING_ROUTE = "/consent/reporting";
-export const PROPERTIES_ROUTE = "/consent/properties";
-export const ADD_PROPERTY_ROUTE = "/consent/properties/add-property";
 
 // Management group
```

Comment:
Should the "Management" routes be updated to be "Settings" instead?

Reply:
Updated!